src/workerd/server/tests/python/langchain/langchain.wd-test

```diff
@@ -0,0 +1,19 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "python-hello",
+      worker = (
+        modules = [
+          (name = "worker.py", pythonModule = embed "./worker.py"),
+          (name = "aiohttp", pythonRequirement = "aiohttp"),
+          (name = "ssl", pythonRequirement = "ssl"),
+          (name = "langchain==0.0.339", pythonRequirement = ""),
+          (name = "openai==0.28.1", pythonRequirement = ""),
```

Comment:
```suggestion
          (name = "langchain", pythonRequirement = ""),
          (name = "openai", pythonRequirement = ""),
```

Reply:
I think this breaks it doesn't it?

Reply:
I suppose in workerd it does. But then we should pin the requirements in workerd's code, not here. If we don't then workerd will have different behaviour to edgeworker (since in edgeworker it is pinned via the packages we embed)

Reply:
Well I just wanted to get some tests into workerd so that I have a place to stick more things. I agree that this should be fixed.